### ISSUE TITLE
Communication Wizard Markup prevents media query use 

### ISSUE THREAD

**Issue by garrettjohnson** [SparkDevNetwork/Rock#2800]
### Prerequisites
* [x] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

Bug/Enhancement

The communication wizard uses inline width definitions since there is no way to guarantee proper display on user created templates without it ie `<table width='100%'><tr><td width='50%' valign='top'>`

However, this markup prevents the use of media queries to change the display based on viewport size. I'd propose adding an option to disable the inline width, and as well as adding CSS classes to assist with styling. Since Zurb's Foundation for email appears to be the underlying framework for the default template:

- `<table width='100%'>` would become `<table class="row">`
- `<td width='50%' valign='top'><div class='dropzone'>` would become `<td class='dropzone columns large-6 small-6 first'>`

Example 2-Column Markup:

```
<table class='row'><tr><td class='dropzone columns large-6 small-6 first' valign='top'></td><td class='dropzone columns large-6 small-6 last'></td></tr></table>
```

### Versions

* **Rock Version:** 7.2  
* **Client Culture Setting:** en-US

**Comment by edmistj** [comment]
@garrettjohnson have you tested this in multiple email clients?

**Comment by garrettjohnson** [comment]
Yes. I've ran the Zurb markup through Litmus ([Preview](https://litmus.com/checklist/emails/public/8f4937b)/[Code](https://litmus.com/builder/93306a4)). I understand why the current markup exists; you can't guarantee people are going to include the required CSS in their email template.

While playing with [the existing markup](https://codepen.io/garrettjohnson/pen/aqYzvV) I realized a CSS `width` property will take precedence over a `<table width="">` property. So, it's possible to override the widths and achieve responsiveness by just adding classes to assist with targeting.

**Comment by edmistj** [comment]
OK can you provide the new template markup. We'll create a migration to update it assuming that someone hasn't already done a local change on their side.

**Comment by garrettjohnson** [comment]
Submitted a PR for the toolbar; the included sidebar template is already using custom markup for it's sidebar so it's unaffected. (Since it `<td style="width: 3%"> </td>` to achieve padding between the main content and the sidebar.) Happy to rewrite it if you want!

**Comment by edmistj** [comment]
Thanks! I think that one is fine.
